lib/Locale/Country/Extra.pm

```diff
@@ -24,9 +25,11 @@ our %COUNTRY_MAP = (
     "south georgia"                     => "gs",
     "south georgia & south sandwich"    => "gs",
     "syrian arab republic"              => "sy",
+    "taiwan"                            => "tw",
     "u.a.e."                            => "ae",
     "vatican city state"                => "va",
     "virgin islands"                    => "vg"
+
```

Comment:
Why the extra line? Plus wouldn't hurt to add a comma to the last line as well so github doesn't pick it up as a change if we add another country below it.